torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -125,7 +170,7 @@ struct GraphFuser {
         }
       #endif
     }
-    return false;
+    return allow_constant_numbers && node->type()->isSubtypeOf(NumberType::get());
```

Comment:
You didn't check that the number is in fact a constant. Even if it's not a bug, we should at least change the name of the argument to `allow_numbers`.